<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Chat</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>131</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Users Online:</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>171</width>
          <height>461</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>171</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textWindow">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>651</width>
          <height>391</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>316</width>
          <height>181</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="sendMessageContent">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>470</y>
          <width>561</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sendDataButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>470</y>
          <width>85</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Droid Sans</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>440</y>
          <width>851</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Write to File</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Connect</string>
       </attribute>
       <widget class="QLineEdit" name="usernameField">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>421</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ipField">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>240</y>
          <width>301</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="portField">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>240</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>241</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter your username</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>210</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IP Address</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>210</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>300</y>
          <width>421</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
